pkg/kubelet/dockershim/docker_sandbox_test.go

```diff
@@ -154,6 +155,31 @@ func TestSandboxStatus(t *testing.T) {
 	assert.Error(t, err, fmt.Sprintf("status of sandbox: %+v", statusResp))
 }
 
+// TestSandboxHasLeastPrivilegesConfig tests that the sandbox is set with no-new-privileges
+// and it has a seccomp profile.
+func TestSandboxHasLeastPrivilegesConfig(t *testing.T) {
+	ds, _, _ := newTestDockerService()
+	config := makeSandboxConfig("foo", "bar", "1", 0)
+	ds.podSandBoxSeccomp = "{defaultAction=\"SCMP_ACT_ERRNO\"}"
```

Comment:
How about using backticks?
```suggestion
	ds.podSandBoxSeccomp = `{defaultAction="SCMP_ACT_ERRNO"}`
```

Reply:
Thank you reviewing the changes, @php-coder, I have replaced the interpreted string with a raw string and removed all the escaped quotes.